app/renderer/js/pages/preference/general-section.js

```diff
@@ -76,10 +78,21 @@ class GeneralSection extends BaseSection {
 						<div class="setting-description">Always start minimized</div>
 						<div class="setting-control"></div>
 					</div>
+				</div>
+				<div class="title">Spellchecker (requires restart)</div>
+				<div class="settings-card">
 					<div class="setting-row" id="enable-spellchecker-option">
-						<div class="setting-description">Enable spellchecker (requires restart)</div>
+						<div class="setting-description">Enable spellchecker</div>
```

Comment:
Probably add `(auto-detect language)` or something like to that to this statement.

Reply:
Hmm, I don't think we need to add something like that since in terms of UX. And if we feel strongly about adding something I think "(language is auto-detected)" is better wording for this.

Reply:
Okay, it's probably better to say more than to say nothing and avoid confusion, so I'll add spellchecker (language is auto-detected by default) for now.